Amazon-A2I-Rekognition-Custom-Label-notebook.ipynb

```diff
@@ -179,7 +190,7 @@
    "outputs": [],
    "source": [
     "#Replace the project-arn below with the project-arn of your project from the describe-projects output above\n",
-    "!aws rekognition describe-project-versions --project-arn <your-project-arn>"
+    "!aws rekognition describe-project-versions --project-arn 'arn:aws:rekognition:us-east-1:841408598787:project/Bl5v2-rekog-a2i-project/1600705521346'"
```

Comment:
we generally don't include our own resource arns and account ids in the code samples but prefer to have either variables that will automatically populate it or instruct the customer to put theirs in 

this feedback applies to other areas in this commit as well